index.html

```diff
@@ -1401,9 +1401,47 @@ <h2>Core Vocabulary Definitions</h2>
 <p>
      <span class="rfc2119-assertion" id="td-context-default-language">
      In the <code>@context</code> array, the optional default language that is 
-     used throughout a TD document is defined within an object as a member with 
+     used throughout a TD document for human-readable strings outside of a <a href="#multilanguage">MultiLanguage</a> object
+     is also defined within an object as a member with 
      a key <code>@language</code> whose value MUST be a string representing a [[!BCP47]] 
      language code.</span>
+     <span class="rfc2119-assertion" id="td-context-default-language-last">
+     The object used to define the default language MUST be the last object in
+     the `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-unique">
+     There MUST be only one object defining a default language in an `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-separate">
+     The object used to define the default language MUST NOT be combined with any other
+     objects defining context extensions (additional vocabulary terms).
+     </span>
+</p>
+<p>
+     Identification of the base direction of human-readable text in TDs 
+     is based on the following set of rules:
+     <ul>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-inference">
+     The base text direction MUST be inferred from the default language.
+     </span></li>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-inference">
+     In cases where a language can be written in more than one script which use 
+     different writing directions, the value used for `@language` MUST
+     include a script subtag so that an appropriate base direction can be inferred. 
+     </span></li>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-default">
+     If no default language is given, the base text direction MUST be assumed to be LTR (left-to-right).
+     </span>  This implies that if the language used in human readable text uses a script
+     that is written RTL (right-to-left), the default language needs to be explicitly specified.</li>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-independence">
+     Each human-readable string MUST be processed independently, starting from the base
+     direction defined for the TD.</span></li>
```

Comment:
Where does this come from? Does it mean the direction has to be computed every time, although the default language cannot have changed?

Reply:
No, it means you can't carry forward changes in direction from one string to another, or take the order that strings in the document appear as part of the processing.   Each string is processed independently, starting from the same base direction.  Which is what I said.

Reply:
This actually relates to some points in the internationalization questionnaire.   Anyway, it seems obvious, but I added it to make SURE that no one does this.   You could imagine someone doing something silly like processing the entire JSON document as an string...

Reply:
I easily understood the explanation with "can't carry forward changes in direction from one string to another", opposed to the assertion text... maybe we can improve accordingly

Reply:
I easily understood the explanation with "can't carry forward changes in direction from one string to another", opposed to the assertion text... maybe we can improve accordingly.

This should go to Section 6.

Reply:
I reworded the assertion a little but, but also added an "In other words..." statement after it to clarify.  Also moved to Section 6 since this is a serialization (or rather, a deserialization) concern.